File: C/menor_de_tres.c
/*
 ***********************************************************
 * Descrição:
 * Fazer um programa para ler três números inteiros. Em seguida,
 * mostrar qual o menor dentre os três números lidos. Em caso
 * de empate, mostrar apenas uma vez.
 ***********************************************************
 */

 #include <stdio.h>

 int main() {

    int a, b, c, menor;

    printf("Primeiro valor: ");
    scanf("%d", &a);
    printf("Segundo valor: ");
    scanf("%d", &b);
    printf("Terceiro valor: ");
    scanf("%d", &c);

    menor = a;

    if (b < menor) {
        menor = b;
    }

    if (c < menor) {
        menor = c;
    }

    printf("MENOR = %d\n", menor);

    return 0;
 }

File: Python/menor_de_tres.py
#  ***********************************************************
#  * Descrição:
#  * Fazer um programa para ler três números inteiros. Em seguida,
#  * mostrar qual o menor dentre os três números lidos. Em caso
#  * de empate, mostrar apenas uma vez.
#  ***********************************************************

a = int(input("Primeiro valor: "))
b = int(input("Segundo valor: "))
c = int(input("Terceiro valor: "))

menor = a

if b < menor:
    menor = b
if c < menor:
    menor = c

print(f"Menor = {menor}")
